contrib/libucl/include/ucl++.h

```diff
@@ -396,6 +396,11 @@ class Ucl final {
 		return default_val;
 	}
 
+	std::string forced_string_value () const
+	{
```

Comment:
Looks like we'd need to update the style checker for C++

Reply:
It would be nice if these changes were expressible in a .clang-format configuration file to allow for easy conformance formatting.

Reply:
Can clang-format apply different settings for C and C++, from the same .clang-format file?

Reply:
@emaste I think the the `Language:` key should do that. See https://clang.llvm.org/docs/ClangFormatStyleOptions.html